pymc/distributions/shape_utils.py

```diff
@@ -97,6 +97,12 @@ def _check_shape_type(shape):
 StrongDimsWithEllipsis: TypeAlias = Sequence[str | EllipsisType]
 StrongSize: TypeAlias = TensorVariable | tuple[int | Variable, ...]
 
+CoordValue: TypeAlias = Sequence[Hashable] | np.ndarray | None
+Coords: TypeAlias = Mapping[str, CoordValue]
+
+StrongCoordValue: TypeAlias = tuple[Hashable, ...] | None
+StrongCoords: TypeAlias = Mapping[str, StrongCoordValue]
```

Comment:
Put these in a more independent place to avoid the new circular import issues. Coordinates are not really distribution specific anyway